src/System.Runtime.Intrinsics/src/System/Runtime/Intrinsics/Types.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    // 128 bit types
+    [StructLayout(LayoutKind.Sequential, Size = 16)]
+    public struct Vector128<T> where T : struct 
```

Comment:
@jkotas These intrinsic APIs will be implemented in S.P.CoreLib. Do you mean that we do not need all the files under `System.Runtime.Intrinsics/src/` and `System.Runtime.Intrinsics.X86/src/` in CoreFX?